src/send-request.ts

```diff
@@ -117,11 +123,7 @@ export const xhr = ({
                 }
 
                 if (callback) {
-                    if (options.verbose) {
-                        callback({ status: 0, error: error })
-                    } else {
-                        callback(0)
-                    }
+                    callback({ status: 0 })
```

Comment:
This fixes feature flags sending `[]` when decide errors.

https://github.com/PostHog/posthog-js/pull/420/files#diff-ddd24b8ce37c6ae2630e5b9f5a3a492bc0f16beabfa27858c3e19ed43a97d67c

🙈